src/pages/register-a-beacon/about-the-vessel.tsx

```diff
@@ -70,81 +62,43 @@ const AboutTheVessel: FunctionComponent<FormPageProps> = ({
 }: FormPageProps): JSX.Element => {
   const pageHeading = "About the vessel, windfarm or rig/platform";
 
-  const backButton: ReactNode = (
-    <BackButtonRouterIndexes href="/register-a-beacon/activity" />
+  const pageText: ReactNode = (
+    <InsetText>
+      Leave anything that isn&apos;t relevant blank. Any information you do
+      provide may help save lives in a Search and Rescue scenario.
+    </InsetText>
   );
 
   return (
-    <>
-      <Layout
-        navigation={backButton}
-        title={pageHeading}
-        pageHasErrors={form.hasErrors}
-        showCookieBanner={showCookieBanner}
-      >
-        <Grid
-          mainContent={
-            <>
-              <FormErrorSummary formErrors={form.errorSummary} />
-              <Form>
-                <FormFieldset>
-                  <FormLegendPageHeading>{pageHeading}</FormLegendPageHeading>
-
-                  <InsetText>
-                    Leave anything that isn&apos;t relevant blank. Any
-                    information you do provide may help save lives in a Search
-                    and Rescue scenario.
-                  </InsetText>
-
-                  <MaxCapacityInput
-                    value={form.fields.maxCapacity.value}
-                    errorMessages={form.fields.maxCapacity.errorMessages}
-                  />
-
-                  <VesselNameInput value={form.fields.vesselName.value} />
-
-                  <BeaconLocationInput
-                    value={form.fields.beaconLocation.value}
-                    errorMessages={form.fields.beaconLocation.errorMessages}
-                  />
-
-                  <SectionHeading>Vessel information</SectionHeading>
-
-                  <PortLetterNumberInput
-                    value={form.fields.portLetterNumber.value}
-                  />
-
-                  <HomeportInput value={form.fields.homeport.value} />
-
-                  <AreaOfOperationTextArea
-                    value={form.fields.areaOfOperation.value}
-                    errorMessages={form.fields.areaOfOperation.errorMessages}
-                  />
-
-                  <ImoNumberInput value={form.fields.imoNumber.value} />
-
-                  <SsrNumberInput value={form.fields.ssrNumber.value} />
-
-                  <OfficialNumberInput
-                    value={form.fields.officialNumber.value}
-                  />
-
-                  <SectionHeading>
-                    Windfarm, rig or platform information
-                  </SectionHeading>
-
-                  <RigPlatformLocationInput
-                    value={form.fields.rigPlatformLocation.value}
-                  />
-                </FormFieldset>
-                <Button buttonText="Continue" />
-              </Form>
-              <IfYouNeedHelp />
-            </>
-          }
-        />
-      </Layout>
-    </>
+    <BeaconsForm
```

Comment:
Updated to use the `BeaconsForm` component as there were issues with the hidden form inputs not being on the page with using the `Layout`/`Grid` components